packages/react-ui-core/src/Ratings/Star.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import LinearGradient from './LinearGradient'
+
+export default class Star extends Component {
+  static propTypes = {
+    id: PropTypes.string,
```

Comment:
If we are going to keep having the parent pass in this `id` prop it needs to be required.